app/controllers/planning_applications_controller.rb

```diff
@@ -54,7 +54,7 @@ def create
     if @planning_application.save
       audit("created", nil, current_user.name)
       flash[:notice] = "Planning application was successfully created."
-      receipt_notice_mail if @planning_application.applicant_email.present?
+      receipt_notice_mail if @planning_application.agent_email.present? || @planning_application.applicant_email.present?
```

Comment:
this could be `email_agent_and_applicant.any?`

Reply:
but our email_agent_and_applicant method is inside  PlanningApplicationMailer - how would the controller know about this method given it doesn't belong to the planningapplication itself? (all the emails are being called by redeclaring the methods inside the controllers, like so:
```  def receipt_notice_mail
    PlanningApplicationMailer.receipt_notice_mail(
      @planning_application,
      request.host,
    ).deliver_now
  end```

how does one go around that?

Reply:
sorry I didn't realise it was in a controller; this is probably trying to tell us the logic should be on the model but I'm happy to leave it as such for the time being.